core/src/test/scala/kafka/metrics/ClientInstanceSelectorTest.scala

```diff
@@ -0,0 +1,55 @@
+package kafka.metrics
+
+import kafka.metrics.clientmetrics.CmClientInformation
+import org.apache.kafka.common.Uuid
+import org.apache.kafka.common.errors.InvalidConfigurationException
+import org.junit.jupiter.api.Assertions.{assertFalse, assertThrows, assertTrue}
+import org.junit.jupiter.api.{Test, Timeout}
+
+@Timeout(120)
+class ClientInstanceSelectorTest {
+
+  def createClientInstanceSelector(): CmClientInformation = {
+    val client_instance_id = Uuid.randomUuid().toString
+    val clientId = "testclient1"
+    val softwareName = "Confluent.Java"
+    val softwareVersion = "89.2.0"
+    val hostAddress = "1.2.3.4"
+    val port = "9092"
```

Comment:
Not relevant here really, but this looks more like the broker port than the client port and may thus be confusing (the port here is the client's port)

Reply:
I will rename the variable to client port here and in other places too.

Reply:
BTW do you know how I can get the client port from the client request?